File: app/src/main/java/com/instaconnect/android/utils/models/Response.java
package com.instaconnect.android.utils.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Response{

	@SerializedName("code")
	private String code;

	@SerializedName("messagelist")
	private List<MessagelistItem> messagelist;

	@SerializedName("is_last_page")
	private Integer isLastPage;

	@SerializedName("message")
	private String message;

	public void setCode(String code){
		this.code = code;
	}

	public String getCode(){
		return code;
	}

	public void setMessagelist(List<MessagelistItem> messagelist){
		this.messagelist = messagelist;
	}

	public List<MessagelistItem> getMessagelist(){
		return messagelist;
	}

	public void setIsLastPage(Integer isLastPage){
		this.isLastPage = isLastPage;
	}

	public Integer getIsLastPage(){
		return isLastPage;
	}

	public void setMessage(String message){
		this.message = message;
	}

	public String getMessage(){
		return message;
	}
}

File: app/src/main/java/com/instaconnect/android/ui/trending_websites/models/Response.kt
package com.instaconnect.android.ui.trending_websites.models

import com.google.gson.annotations.SerializedName
import com.instaconnect.android.ui.trending_websites.models.Streamingwebsites
import com.instaconnect.android.ui.trending_websites.models.WebsitesItem

class Response {
    @SerializedName("code")
    var code: String? = null

    @SerializedName("streamingwebsites")
    var streamingwebsites: Streamingwebsites? = null

    @SerializedName("message")
    var message: String? = null
}